src/main/java/aQute/jpm/platform/Platform.java

```diff
@@ -146,6 +146,9 @@ public String defaultCacertsPassword() {
 	protected void process(String resource, CommandData data, String path, Map<String,String> map, String... extra)
 			throws Exception {
 		File file = new File(path);
+		if(file.exists()) {
```

Comment:
hey @Seiphon  this will short-circuit the entire 'process' method.  I am afraid this will lead to bugs where some other part of the 'install' process needs to complete. Could you investigate if we simply can merge the incoming 'map' with the existing arguments?

Reply:
This method uses a class called "Sed" which is about updating an existing file.  So it seems we should be able to update correctly instead of just returning early.

Reply:
Hi @gamerson, when I debugging, I found the incoming "map" always be null. I will try to read these arguments from pre file and merge them with new arguments.

Reply:
I will close this pr and resend one.